elastic/kibana
# [Discover][SavedSearch] Fix default rowsPerPage for Dashboard panels

- Follow up for https://github.com/elastic/kibana/pull/180536

## Summary

This PR fixes an issue with `rowsPerPage` Advanced Setting: after the refactoring it was ignored. 

To test: 
Change `rowsPerPage` on Advanced Setting page, navigate to Dashboard and add a saved search panel. It should use the configured value by default. The default value can be overwritten by custom panel settings or saved search own settings.


### Checklist
